cdap-app-fabric/src/main/java/io/cdap/cdap/internal/app/store/DefaultStore.java

```diff
@@ -594,12 +594,19 @@ public void markApplicationsLatest(Collection<ApplicationId> appIds)
   }
 
   @Override
-  public int addApplication(ApplicationId id, ApplicationMeta meta) throws ConflictException {
+  public int addLatestApplication(ApplicationId id, ApplicationMeta meta) throws ConflictException {
     return TransactionRunners.run(transactionRunner, context -> {
       return getAppMetadataStore(context).createApplicationVersion(id, meta);
```

Comment:
nit: we should directly pass true here. No need for overloading this method

Reply:
Followed the same pattern to rename `createApplicationVersion(appid, appmeta)` to `createLatestApplicationVersion` to avoid overloading and defining the behaviour (of the overloaded method) with a boolean param.